addons/cert.py

```diff
@@ -242,7 +242,8 @@ def int31(data, platform):
 def env33(data):
     for token in data.tokenlist:
         if isFunctionCall(token, ('system',), 1):
-            reportError(token, 'style', 'Do not call system()', 'ENV33-C')
+            if not simpleMatch(token, "system ( NULL )") and not simpleMatch(token, "system ( 0 )"):
```

Comment:
I would have used the valueflow. Something like;

    argValue = token.next.astOperand2.getValue(0)
    if argValue and argValue.intvalue == 0 and argValue.isKnown():

That would avoid FPs if there is a different null pointer constant used or if there is a cast etc..